package.json

```diff
@@ -39,10 +39,22 @@
       }
     }
   },
+  "mocha": {
```

Comment:
The mocha config here might be mistakenly merged into this branch right? It was already moved to a separate file.